vformer/attention/spatial.py

```diff
@@ -69,6 +69,20 @@ def __init__(
             self.sr = nn.Conv2d(dim, dim, kernel_size=1, stride=1)
 
     def forward(self, x, H, W):
+        """
+
+        Args:
```

Comment:
We are following NumPy-style, so this needs to be the following way, doesn't it?

```suggestion
        Parameters
        --------------
```